originals/h.yaml

```
@@ -222,6 +222,24 @@
     genres:
     - Real-Time Strategy
 
+- name: 'Higurashi: When They Cry Hou'
+  external:
+    website: 'https://entergram.co.jp/higurashihou/'
```

```suggestion
    website: 'https://entergram.co.jp/higurashihou/'
    wikipedia: Higurashi When They Cry
```
The wikipedia link is preferred as it is in english